elixir/.gitignore

```diff
@@ -0,0 +1,23 @@
+# The directory Mix will write compiled artifacts to.
+/_build
+
+# If you run "mix test --cover", coverage assets end up here.
+/cover
```

Comment:
Can we add some of these to .dockerignore as well, just for completeness? I think `/cover`, `/doc`, and `erl_crash.dump` definitely. (I've not seen `/.fetch` before so I'm not sure what it actually does, but maybe it also should be ignored for deployment?)